pedometer/chapter.md

```diff
@@ -11,77 +13,79 @@ Let's roll up our sleeves, and prepare to untangle.
 
-A person bounces up and down, in the vertical direction, with each step. If you are walking on Earth, or another big ball of mass floating in space, the bounce is conveniently in the same direction as gravity.
+A person bounces up and down, in the vertical direction, with each step. If you are walking on Earth (or another big ball of mass floating in space) the bounce is conveniently in the same direction as gravity.
 
-**We are going to count steps by using the accelerometer to count bounces up and down. Because the phone can rotate in any direction, we will take advantage of gravity to know which direction down is. A pedometer can count steps by counting the number of bounces in the direction of gravity.**
+We are going to count steps by using the accelerometer to count bounces up and down. Because the phone can rotate in any direction, we will take advantage of gravity to know which direction down is. A pedometer can count steps by counting the number of bounces in the direction of gravity.
 
-Let's look at a person walking with a smartphone containing an accelerometer, in his or her shirt pocket, as depicted below.
+Let's look at a person walking with an accelerometer-equipped smartphone equipped in his or her shirt pocket, as depicted below.
 
-![](chapter-figures/walk-1.png)\
+![](chapter-figures/walk-1.png)
 
 For the sake of simplicity, we'll assume that the stick person:
 
-* is walking in the z direction;
-* bounces with each step in the y direction;
-* maintains the phone in the same orientation throughout the entire walk;
+* is walking in the $z$ direction;
+* bounces with each step in the $y$ direction; and
+* maintains the phone in the same orientation throughout the entire walk.
 
-In our perfect world, acceleration from step bounces will form a perfect sine wave in the y direction. Each peak in the sine wave is exactly one step. Step counting becomes a matter of counting these perfect peaks.
+In our perfect world, acceleration from step bounces will form a perfect sine wave in the $y$ direction. Each peak in the sine wave is exactly one step. Step counting becomes a matter of counting these perfect peaks.
 
 Ah, the joys of a perfect world, which we only ever experience in texts like this. Don't fret, things are about to get a little messier, and a lot more exciting. Let's add a little more reality to our world.
 
 ## Even Perfect Worlds Have Fundamental Forces of Nature
 
-The force of gravity causes an acceleration in the direction of gravity, which we refer to as gravitational acceleration. This acceleration is unique because it is always present, and is always constant at $9.8 m/s^2$.
+The force of gravity causes an acceleration in the direction of gravity, which we refer to as gravitational acceleration. This acceleration is unique because it is always present, and is constant at 9.8 $m/s^2$.
```

Comment:
That's fair enough. I didn't miss any of the other clarifications you removed, so that's awesome. But I can hear the "actually" brigade in my head about this one. "Actually, it's not constant..." You know how they are...

Maybe we can change it to, "is always present and, for the purposes of this chapter, is constant..."? Surely @MichaelDiBernardo won't begrudge us six words? :)